src/sema/expression.rs

```diff
@@ -30,83 +30,6 @@ use crate::Target;
 use base58::{FromBase58, FromBase58Error};
 
 impl Expression {
-    /// Return the location for this expression
```

Comment:
Why was this moved?

Reply:
`fn loc` is a very essential method for `Expression`. I thought it would be nicer to move it closer to the `struct Expression` declaration to ease development and organize the code better.

Reply:
Good point